src/consensus/pbft/pbft_new_views.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "consensus/pbft/libbyz/parameters.h"
+#include "node/entities.h"
+
+#include <array>
+#include <msgpack/msgpack.hpp>
+#include <vector>
+
+namespace pbft
+{
+  struct NewView
+  {
+    int64_t view;
```

Comment:
`View view`? `View` seems to be defined in `libbyz/types.h`.

Reply:
See the other files `pbft_pre_prepares.h` and `pbft_requests.h`. `libbyz/types.h` includes `libbyz/pbft_types.h` which in turn includes `pbft_new_views.h` causing a circular dependency if we do this. This needs to be cleaned up and dealt with, but is outside the scope if this PR.